extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -153,4 +153,10 @@
      */
     @ConfigItem(defaultValue = "jib-image.id")
     public String imageIdFile;
+
+    /**
+     * Whether or not to operate offline.
+     */
+    @ConfigItem
```

Comment:
is the default false?

Reply:
I think so. I actually had a look at an existing flag in the same area: https://github.com/quarkusio/quarkus/blob/main/extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/ContainerImageConfig.java#L66

But it's probably better to make it explicit like here: https://github.com/quarkusio/quarkus/blob/main/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java#L124

I'll push an update.

Reply:
Just to be clear, I was asking about the default JIB behaviour is this flag is not. If it true, it probably make sense to make our default `true` too, to not "break" existing setup.

Reply:
Ah sorry. It's disabled by default.

Reply:
Perfect, thanks.